### PR TITLE
Fixed logging bug for object detection with validation set

### PR DESCRIPTION
**Pull request by joihn**
## Motivation

for object detection, the validation is done in 2 steps:
- validation bounding box Ap metric 
- validation Loss
the problem is that they are reported *separately* to wandb, with the *same* iter number, so wandb will ignore the second one (loss metric ) (undesired).

## Modification

This fix increment the iter number when reporting validation loss

## BC-breaking (Optional)

My fix might not be the best approach if the user is not using this lib for object detection, but should still work (untested in this case) (I tested only with MMdetection)

### CODE REVIEW

**Comment by MengzhangLI** [comment]
Hi, related issue in MMSegmentation: https://github.com/open-mmlab/mmsegmentation/issues/1068.

Our user says it would not fix bug in MMSegmentation but he or she gave the alternative solution.

Best,

File: mmcv/runner/hooks/logger/wandb.py [hunk 1]
```diff
@@ -45,8 +45,20 @@ def log(self, runner):
         tags = self.get_loggable_tags(runner)
         if tags:
```
Comment (fcakyon) [inline mmcv/runner/hooks/logger/wandb.py:46]: @joihn @ZwwWayne @MengzhangLI should be:

```python
tags = self.get_loggable_tags(runner)
if tags:
    tags['global_step'] = self.get_iter(runner)
    self.wandb.log(tags, commit=self.commit)
```

check open-mmlab/mmsegmentation#1068.

**Review by fcakyon — CHANGES_REQUESTED** [review]

**Comment by OpenMMLab-Assistant005** [comment]
Hi @joihn ！We are grateful for your efforts in helping improve this open-source project during your personal time. 
Welcome to join OpenMMLab Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤
